joomla/joomla-cms
# [5.1] Allow custom fields form be manipulates like category form

### Summary of Changes
Allow a component to manipulate the custom field form like you [can do with category view](https://github.com/joomla/joomla-cms/blob/5.1-dev/administrator/components/com_categories/src/Model/CategoryModel.php#L431-L460)


### Testing Instructions
- Apply patch
- Open a field form from com_content in the backend
- Check that you get the form [here](https://github.com/joomla/joomla-cms/blob/5.1-dev/administrator/components/com_content/src/Extension/ContentComponent.php#L371) and can manipulate it.

```$form->getName()``` should give ```com_fields.field.com_content.article```

```$form->removeField('display', 'params');``` removes the "Automatic Display" field


### Actual result BEFORE applying this Pull Request
No way to manipulate the form via PHP directly


### Expected result AFTER applying this Pull Request
You can manipulate the form
